frontend/src/app/accountCreation/SecurityQuestion/SecurityQuestion.test.tsx

```diff
@@ -1,11 +1,30 @@
 import { render, screen, fireEvent } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import { act } from "react-dom/test-utils";
+import { MemoryRouter, Route } from "react-router";
 
 import { SecurityQuestion } from "./SecurityQuestion";
 
+jest.mock("../AccountCreationApiService", () => ({
+  AccountCreationApi: {
+    setRecoveryQuestion: (recoveryQuestion: string, recoveryAnswer: string) => {
```

Comment:
Can you add a failure path? Right now I'm not sure it's handled in the app.

Reply:
Added!